src/auto_splitting/mod.rs

```diff
@@ -944,9 +945,15 @@ fn watchdog<T: event::CommandSink + TimerQuery + Send>(shared_state: Arc<SharedS
                 };
 
                 if result.timed_out() {
+                    if !has_timed_out {
+                        log::error!(target: "Auto Splitter", "timeout, no update in {} seconds", TIMEOUT.as_secs_f32());
+                        has_timed_out = true;
+                    }
                     if let Some(auto_splitter) = &*shared_state.auto_splitter.load() {
                         auto_splitter.interrupt_handle().interrupt();
                     }
+                } else {
+                    has_timed_out = false;
```

Comment:
I'm not sure this is the right place. I think this should be set in the Unloaded case, so it doesn't get set all the time.

Reply:
Nvm, the reload case in the other thread never puts the watchdog into the unloaded state, so I guess this is fine.